hi-ml-azure/src/health_azure/himl.py

```diff
@@ -429,20 +478,61 @@ def submit_run_v2(workspace: Optional[Workspace],
     else:
         output_datasets_v2 = {}
 
-    command_job = command(
-        code=str(snapshot_root_directory),
-        command=cmd,
-        inputs=input_datasets_v2,
-        outputs=output_datasets_v2,
-        environment=environment.name + "@latest",
-        compute=compute_target,
-        experiment_name=experiment_name,
-        environment_variables={
-            "JOB_EXECUTION_MODE": "Basic",
-            "AZUREML_COMPUTE_USE_COMMON_RUNTIME": "true"
-        }
-    )
-    returned_job = ml_client.jobs.create_or_update(command_job)
+    job_to_submit: Union[Command, Sweep]
+
+    if hyperparam_args:
+        param_sampling = hyperparam_args[PARAM_SAMPLING_ARG]
+
+        for sample_param, choices in param_sampling.items():
+            input_datasets_v2[sample_param] = choices.values[0]
+            cmd += f" --{sample_param}=" + "${{inputs." + sample_param + "}}"
+
+        command_job = command(
+            code=str(snapshot_root_directory),
+            command=cmd,
+            inputs=input_datasets_v2,
+            outputs=output_datasets_v2,
+            environment=environment.name + "@latest",
+            compute=compute_target,
+            experiment_name=experiment_name,
+            environment_variables={
+                "JOB_EXECUTION_MODE": "Basic",
+                "AZUREML_COMPUTE_USE_COMMON_RUNTIME": "true"
```

Comment:
Do we really need that AZUREML_COMPUTE_USE_COMMON_RUNTIME anymore? here and in line 531